zstd_compression_format.md

```diff
@@ -97,6 +97,42 @@ to decode all concatenated frames in their sequential order,
 delivering the final decompressed result as if it was a single content.
 
 
+Skippable Frames
```

Comment:
I've read the modified document, 
and I agree this section it not too long,
so it should not confuse the reader before they reach `General Structure of Zstandard Frame format`